engines/griffon/cutscenes.cpp

```diff
@@ -234,7 +252,7 @@ void GriffonEngine::intro() {
 			y--;
 		}
 
-		for (int i = 0; i <= 37; i++) {
+		for (int i = 0; i <= 47; i++) {
```

Comment:
Using `for (int i = 0; i < ARRAYSIZE(story); i++) {` would maybe be better - to not duplicate the size of the story array.

Reply:
Thank you for this feedback!  I have modified it as you said, and will proceed to fix the code as you have reviewed them.